src/ruby/yast/path.rb

```diff
@@ -21,56 +21,57 @@ def clone
     end
 
     # concats path
-    def + another
-      another = self.class.from_string(another) unless another.is_a? Yast::Path
-      return another.clone if components.empty?
-      return clone if another.empty?
-      return Path.new(self.to_s+another.to_s)
+    def +(other)
+      other = self.class.from_string(other) unless other.is_a? Yast::Path
+      return other.clone if components.empty?
+      return clone if other.empty?
+      Path.new(to_s + other.to_s)
     end
 
     def to_s
-      '.'+components.join('.')
+      "." + components.join(".")
     end
 
     # gets number of elements
     def size
-      return components.size
+      components.size
     end
 
     # Detect if there is no  elements
     def empty?
-      return components.empty?
+      components.empty?
     end
 
     def <=>(other)
       return nil unless other.is_a? self.class
-      0.upto(size-1) do |i|
+      0.upto(size - 1) do |i|
         return 1 unless other.send(:components)[i]
-        #we strip enclosing quotes for complex expression
-        our_component = components[i].sub(/\A"(.*)"\Z/,"\\1");
-        other_component = other.send(:components)[i].sub(/\A"(.*)"\Z/,"\\1");
+        # we strip enclosing quotes for complex expression
+        our_component = components[i].sub(/\A"(.*)"\Z/, "\\1")
+        other_component = other.send(:components)[i].sub(/\A"(.*)"\Z/, "\\1")
         res = our_component <=> other_component
         return res if res != 0
       end
-      return size <=> other.size
+      size <=> other.size
     end
 
-  private
+    private
```

Comment:
I see you have codified it in yast2: https://github.com/yast/yast-yast2/blob/06efb8a3f824b7d53873192a0d11e9fbf611e8b5/.rubocop.yml#L33
I don't really care about this particular case. Should we adapt https://github.com/yast/yast-devtools/blob/master/ytools/y2tool/rubocop_yast_style.yml ? How does it play with https://github.com/SUSE/style-guides/blob/master/Ruby.md anyway?